spec/rubocop/result_cache_spec.rb

```diff
@@ -113,6 +113,19 @@ def abs(path)
           expect(cache.load[0].status).to eq(:new_status)
         end
       end
+
+      context 'a global offense' do
+        let(:location) { RuboCop::Cop::Offense::NO_LOCATION }
+        let(:offense) do
+          RuboCop::Cop::Offense.new(:warning, location, 'empty file', 'Lint/EmptyFile',
+                                    :unsupported)
+        end
+
+        it 'serializes the range correctly' do
+          cache.save([offense])
+          expect(cache.load[0].location).to eq(location)
+        end
```

Comment:
It seems that the following names could further clarify the intention:

```suggestion
        let(:no_location) { RuboCop::Cop::Offense::NO_LOCATION }
        let(:global_offense) do
          RuboCop::Cop::Offense.new(:warning, no_location, 'empty file', 'Lint/EmptyFile',
                                    :unsupported)
        end

        it 'serializes the range correctly' do
          cache.save([global_offense])
          expect(cache.load[0].location).to eq(no_location)
        end
```